### PR TITLE
Feature/blue icon if active gray if not

### PR DESCRIPTION
**Pull request by AdamNgu**
This actually also contains my changes where the FE requests limited location data on the /find part of the site, I forgot to push and PR that earlier.

### CODE REVIEW

File: src/app/find/mapPage/MapPage.jsx [hunk 1]
```diff
@@ -162,6 +162,7 @@ export default class MapPage extends Component {
       radius: Math.floor(radius),
       minResults,
       searchString,
+      locationFieldsOnly: true,
```
Comment (Rovack) [inline src/app/find/mapPage/MapPage.jsx:165]: Wasn't this change already made in the last PR?

File: src/components/map/LocationMarker.jsx [hunk 2]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import { Marker, InfoWindow } from 'react-google-maps';
 import Button from '../button';
 
+const MARKER_ICON_MAPPINGS = {
+  blue: 'https://www.google.com/maps/vt/icon/name=assets/icons/spotlight/spotlight_pin_v2_shadow-1-small.png,assets/icons/spotlight/spotlight_pin_v2-1-small.png,assets/icons/spotlight/spotlight_pin_v2_dot-1-small.png,assets/icons/spotlight/spotlight_pin_v2_accent-1-small.png&highlight=000000,1400FF,FFFFFF,000000&color=1400FF?scale=1',
```
Comment (Rovack) [inline src/components/map/LocationMarker.jsx:6]: Just to make sure, this is the official way Google exposes these markers? No CDN or anything?

File: src/app/find/mapPage/MapPage.jsx (hunk 1 above)
Comment (AdamNgu) [reply src/app/find/mapPage/MapPage.jsx:165]: I think I forgot to make that PR for the streetlives-web repo :x so here it is

File: src/components/map/LocationMarker.jsx (hunk 2 above)
Comment (AdamNgu) [reply src/components/map/LocationMarker.jsx:6]: Correct, this is the official way to have custom colors on markers

**Pull request merged**